src/System.Net.Primitives/src/System/Net/CookieContainer.cs

```diff
@@ -737,8 +742,8 @@ internal CookieCollection InternalGetCookies(Uri uri)
             int port = uri.Port;
             CookieCollection cookies = null;
 
-            List<string> domainAttributeMatchAnyCookieVariant = new List<string>();
-            List<string> domainAttributeMatchOnlyCookieVariantPlain = null;
+            var domainAttributeMatchAnyCookieVariant = new Collections.Generic.List<string>();
```

Comment:
Using a partial namespace here is a bit strange looking. Why not just leave it as List<string>? If we need namespace qualification, please use the full namespace.